app/src/main/java/com/courseproject/tindar/entities/MessageModel.java

```diff
@@ -0,0 +1,26 @@
+package com.courseproject.tindar.entities;
+
+import java.sql.Timestamp;
+
+/**
+ * Why does this exist? 
+ */
+
+public interface MessageModel {
+    
+    // Getter methods 
+    public abstract String getMessageId();
+    public Timestamp getCreationTime();
+    public String getMessageContent();
+    public String getSentFromId();
+    public String getSentToId();
+
+    // Setter methods--not implemented because they don't seem like they should exist
+    /* 
+    protected void setMessageId(String messageId);
+    protected void setText(String text);
+    protected void setTimestamp(Timestamp timestamp);
+    protected void setSentFrom(String sentFrom);
+    protected void setSentTo(String sentTo);
```

Comment:
are these lines good to remove?

Reply:
possibly. might implement editing messages later. doing logic in the next PR; there's a good chance that we won't need these but I'll handle it after I figure that out.